Documentation/Installation/TutorialDdev.rst

```diff
@@ -0,0 +1,196 @@
+
+            Project Type [php, typo3, ...] (php): typo3
+
+    b.  Or by supplying the parameters (easier to do automatically):
+
+        ..  code-block:: bash
+
+            ddev config  --project-type=typo3 --docroot=public --create-docroot
+
+    project-type
+        Should always be "typo3"
+
+    docroot
+        Is the folder in which all files that have to be reached by
+        the browser. This folder is commonly called :file:`public`.
+
+    create-docroot
+        As the directory does not exist yet allow DDEV to create it for you.
+
+#.  Start the project
+
+    ..  code-block:: bash
+
+        ddev start
+
+    Now the webserver is running but TYPO3 is not installed yet.
+
+#.  Install a basic installation of TYPO3
+
+    ..  code-block:: bash
+
+        ddev composer create "typo3/cms-base-distribution:^11"
+
+    As we just created the project and have no data there yet, answer yes
+    when prompted if it is ok to overwrite most files in this directory.
+
+    ..  note::
+        You now have a **Composer-based TYPO3 installation**. In case anyone
+        asks you.
+
+#.  Run the install tool setup
+
+    .. rst-class:: bignums-warning
+
+    a.  You can do this from console:
+
+        ..  code-block:: bash
+
+            ddev typo3cms install:setup
+
+        When prompted enter a username and password for you main admin user
+        (who is also system maintainer).
+
+        And enter a name for the site that will be displayed in the backend.
+
+        You can set up a very basic site :bash:`site` displaying "Hello World"
+        or just do nothing :bash:`no`.
+
+    b.  Or via GUI:
+
+        #.  Create a file called FIRST_INSTALL in your webroot
+
+            ..  code-block:: bash
+
+                ddev exec touch public/FIRST_INSTALL
+
+        #.  Open the installer
+
+            ..  code-block:: bash
+
+                ddev launch typo3/install.php
+
+#.  Go to the TYPO3 backend:
+
+    ..  code-block:: bash
+
+        ddev launch typo3
+
+    And login with the credentials you just created.
+
+
+What about the database?
+========================
+
+Upon calling :bash:`ddev config` DDEV automatically created a database for
+you. DDEV also created a file :file:`public/typo3conf/AdditionalConfiguration.php`
+in which it stored the credentials for you.
+
+During the installation setup process TYPO3 created all the tables needed by
+default. If you want to have a look at the database, you can find it here:
+
+..  code-block:: bash
+
+    ddev launch -p
+
+What about sending mails?
+=========================
+
+DDEV preconfigured the :file:`public/typo3conf/AdditionalConfiguration.php`
+to fake sending mails. You can see what mails have been send here:
+
+..  code-block:: bash
+
+    ddev launch -m
+
+How do I get rid of that project now?
+=====================================
+
+If you want to all projects from running on DDEV you can call:
```

Comment:
This sentence sounds strange. Maybe: "If you want to stop all DDEV projects ..."